packages/jupyterlab-manager/src/manager.ts

```diff
@@ -420,9 +420,13 @@ class WidgetManager extends ManagerBase<Widget> implements IDisposable {
    */
   get_state_sync(options: IStateOptions = {}) {
       const models = [];
-      for (let model of this._modelsSync.values()) {
-        if (model.comm_live) {
-          models.push(model);
+      // for (let model of this._modelsSync.values()) {
+      for (let k in this._modelsSync.values()) {
+        if (this._modelsSync.values().hasOwnProperty(k)) {
+          let model = (this._modelsSync.values() as any)[k];
+          if (model.comm_live) {
+            models.push(model);
+          }
```

Comment:
Even with `"downlevelIteration": true`, typescript does not seem to properly compile `for .. of` loops to es5.